core/config.go

```diff
@@ -88,3 +93,106 @@ type Config struct {
 	// authentication method
 	PatchBeanstalkUserdata string
 }
+
+// ParseConfig loads configuration from command line flags, environments variables, and config files.
+func ParseConfig(conf *Config) {
+
+	// The use of FlagSet allows us to parse config multiple times, which is useful for unit tests.
+	flagSet := flag.NewFlagSet("AutoSpotting", flag.ExitOnError)
+
+	var region string
+
+	if r := os.Getenv("AWS_REGION"); r != "" {
+		region = r
+	} else {
+		region = endpoints.UsEast1RegionID
+	}
+
+	conf.LogFile = os.Stdout
+	conf.LogFlag = log.Ldate | log.Ltime | log.Lshortfile
+	conf.MainRegion = region
+	conf.SleepMultiplier = 1
+	conf.LicenseType = os.Getenv("LICENSE")
```

Comment:
This looks unnecessary, it should be handled below

Reply:
Thanks, that was copied from the old `parseCommandLineFlags`. Updated.